dark-mode.css

```diff
@@ -297,6 +301,13 @@ body.dark-mode .rc-switch-double {
 	color: var(--color-white);
 }
 
+body.dark-mode .rc-tags__input {
+	color: var(--color-white);
+}
+
+body.dark-mode .rc-popup-list {
```

Comment:
What is this rule for `.rc-popup-list` doing, exactly? Can you show me a screenshot? I ask because I thought this was taken care of with [the variables around line 113](https://github.com/pbaity/rocketchat-dark-mode/pull/17/files#diff-f2e33b3cc2c42b19bde0c0a20119973fR113).

Reply:
I think it is my mistake.
In that fix I added only [that](https://github.com/pbaity/rocketchat-dark-mode/pull/17/commits/fc5fad7464739efbb03c0b23918ccbf78152cfec)
But than I realized, that I was edit my local version of dark-mode.css file with my comments, which you removed earlier. And I removed my comment and did another [push](https://github.com/pbaity/rocketchat-dark-mode/pull/17/commits/d5c4ac17da9ee2cb23175205684017b767bd7298) 
I don't know why 

> body.dark-mode .rc-popup-list {

 line is shown like it was changed.
I think, that I should refresh my local file from upstream before make new changes. And it was my mistake.